Documentation/network/bgp-control-plane.rst

```diff
@@ -78,6 +78,9 @@ The policy in ``yaml`` form is defined below:
        - peerAddress: 'fc00:f853:ccd:e793::50/128'
          peerASN: 64512
          eBGPMultihopTTL: 10
+         connectRetryTime: 120s
+         holdTime: 90s
+         keepAliveTime: 30s
```

Comment:
nit: we should keep values consistent in the example. I mean gracefulRestart.restartTime is in quotes, so either we put all timer values in quotes or remove quotes from restartTime. 

I don't have any strong preference on which way to go.